migrations/V183__Add_GRB_review_members.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TYPE grb_reviewer_role_type AS ENUM (
+  'CO_CHAIR_CIO',
+  'CO_CHAIR_CFO',
+  'CO_CHAIR_HCA',
+  'ACA_3021_REP',
+  'CCIIO_REP',
+  'PROGRAM_OPERATIONS_BDG_CHAIR',
+  'CMCS_REP',
+  'FED_ADMIN_BDG_CHAIR',
+  'PROGRAM_INTEGRITY_BDG_CHAIR',
+  'QIO_REP',
+  'SUBJECT_MATTER_EXPERT',
+  'OTHER'
+);
+
+CREATE TYPE grb_reviewer_voting_role_type AS ENUM (
+  'VOTING',
+  'ALTERNATE',
+  'NON_VOTING'
+);
+
+CREATE TABLE IF NOT EXISTS system_intake_grb_reviewers (
+  id UUID PRIMARY KEY NOT NULL,
+  eua_user_id ZERO_STRING NOT NULL,
```

Comment:
Do you think it makes sense to try and utilize the `user_account` table here instead of just EUA IDs?
```suggestion
  eua_user_id UUID NOT NULL REFERENCES user_account(id),
```

Reply:
I'm not very familiar with the user account implementation, but is it possible to add members that don't have an EASi account or did we automate adding new users?

Reply:
@mynar7, the code should be in there already to create a user account :). We do this in MINT with model plan collaborator. [Code Reference Here](https://github.com/CMSgov/mint-app/blob/main/pkg/graph/resolvers/plan_collaborator.go#L47).

Reply:
OK, so just let the helper create the user account if it doesn't already exist when creating GRB members? I'm still wrapping my head around "user accounts" that are created by a third party, but thanks for the reference! I was looking at some of those same helpers in EASi earlier.

Reply:
Awesome! 

Yeah, that's exactly it! Basically the third party account creation just sets a flag in the table that the user hasn't logged in, and when the user logs in, it updates some of the account information.